docs/osquery/osquery.asciidoc

```diff
@@ -61,12 +61,11 @@ TIP: To save a single query for future use, click *Save for later* and define th
 [[osquery-view-history]]
 == View or rerun previous live queries
 
-The *Live queries history* section on the *Live queries* tab shows a log of queries run over the last 30 days.
-Each query has the following options:
+The *Live queries history* section on the *Live queries* tab shows a log of queries run over the last 30 days. From the Live queries table, you can:
 
-* Click image:images/play-icon.png[Right-pointing triangle] to rerun a query.
+* Click the triangle icon (image:images/play-icon.png[Right-pointing triangle]) to rerun a single query or a query pack.
```

Comment:
˙How about referring to the icon by its function:  Click the run icon ....

Reply:
Thanks for the review, @gchaps! Is this UI element formally named the run icon? I wasn't sure whether it had been officially named yet.

Reply:
Good question.  Does the icon have a tooltip that provides a name?

Reply:
I didn't see a name when I checked the icon in Kibana and Security. The screenshot below is from Kibana. I hovered over the icon to see if a tooltip popped up. It didn't. @james-elastic @patrykkopycinski would one of you mind confirming that the icon/button doesn't have a tooltip defined in the UI?
<img width="118" alt="Screen Shot 2022-09-23 at 4 58 04 PM" src="https://user-images.githubusercontent.com/79124755/192055998-40b5e8c0-d72e-4222-ae94-03c32c2e2c56.png">

Reply:
@james-elastic @patrykkopycinski if a name hasn't been decided on already, I like "run". We already use this verb in the docs when we tell users to execute queries, so it'll align nicely with our current instructions.

Reply:
@nastasha-solomon I think it's bug on our side, there should be a tooltip with `Run <query_name|pack_name>`

Reply:
Adding icon name:

```suggestion
* Click the run icon (image:images/play-icon.png[Right-pointing triangle]) to rerun a single query or a query pack.
```